File: java/pca-model-generator/src/main/java/GraphQLSchema.java
import java.util.List;

public class GraphQLSchema
{
    public GraphQLSchema(){
    }

    /*private Object data;

    public void setData(Object mydata){
        data=mydata;
    }

    public Object getData(){
        return data;
    }*/

    public GraphQLQueryType queryType;
    public Object mutationType;
    public Object subscriptionType;
    public List<GraphQLSchemaType> types;
    public Object directives;
    public Object args;

    /*public void setMutationType(Object mutationType){
     _MutationType = mutationType;
    }

    public Object getMutationType(){
        return _MutationType;
    }*/
}

File: net/src/Sdl.Tridion.Api.Client/GraphQLClient/Schema/GraphQLSchema.cs
using System.Collections.Generic;

namespace Sdl.Tridion.Api.GraphQL.Client.Schema
{    
    public class GraphQLSchema
    {
        public GraphQLQueryType QueryType { get; set; }
        public object MutationType { get; set; }
        public object SubscriptionType { get; set; }
        public List<GraphQLSchemaType> Types { get; set; }
        public object Directives { get; set; }
        public object Args { get; set; }
    }
}
